freeplane/src/main/java/org/freeplane/view/swing/map/overview/MapOverviewConstants.java

```diff
@@ -0,0 +1,21 @@
+package org.freeplane.view.swing.map.overview;
+
+import java.awt.Color;
+
+import org.freeplane.api.LengthUnit;
+import org.freeplane.api.Quantity;
+
+public class MapOverviewConstants {
+    public final static String PROP_MAP_OVERVIEW_VISIBLE = "mapOverviewVisible";
+    public final static String PROP_MAP_OVERVIEW_VISIBLE_FS = "mapOverviewVisible.fullscreen";
+
+    public final static String PROP_MAP_OVERVIEW_PREFIX = "map_overview_";
+    public final static String PROP_MAP_OVERVIEW_ATTACH_POINT = PROP_MAP_OVERVIEW_PREFIX + "attach_point";
+    public final static String PROP_MAP_OVERVIEW_BOUNDS = PROP_MAP_OVERVIEW_PREFIX + "bounds";
+
+    public final static int DEFAULT_SIZE = (int) new Quantity<LengthUnit>(240, LengthUnit.pt).in(LengthUnit.px).value;
```

Comment:
instead of using `(int) ... .in(LengthUnit.px).value` I would rather use `.toBaseUnitsRounded()` because it is exactly what this method should do.

It does not contain word "pixels" because Quantity is not necessarily a length.

Reply:
Will be treated in next commits.